SingularityUI/app/selectors/tasks.es6

```diff
@@ -68,7 +68,7 @@ export const getFilteredTasks = createSelector(
         _.each(tasks, (t) => {
           t.id = id.extract(t);
         });
-        const hostMatch = fuzzy.filter(filter.filterText, tasks, host);
+        const hostMatch = fuzzy.filter(filter.filterText.split('-').join('_'), tasks, host);
```

Comment:
Because I didn't realize I could use a `/g` flag to make the replacement global.